bundle/manifests/ibm-websphere-liberty.clusterserviceversion.yaml

```diff
@@ -329,6 +329,13 @@ spec:
       - description: Probe to determine successful initialization. If specified, other probes are not executed until this completes successfully.
         displayName: Startup Probe
         path: probes.startup
+      - description: The license must be accepted before WebSphere Liberty can be deployed.
```

Comment:
This should also be added to `config/manifests/bases/ibm-websphere-liberty.clusterserviceversion.yaml`, otherwise running `make bundle` will strip this out.